docs/managing_deis/production_deployments.rst

```diff
@@ -53,6 +53,25 @@ Back up data
 
 Backing up data regularly is recommended. See :ref:`backing_up_data` for steps.
 
+Change Registration Mode
+------------------------
+
+Changing the registration process is highly recommended in production.  By default, registrations for a
+new cluster are open to anyone with the proper URL.  Once the admin user has registered on a new cluster,
+it is recommended that you either turn off registrations entirely or enable the adminOnly registration feature.
+
+Turn off registrations:
+
+.. code-block:: console
+
+   $ deisctl config platform set registrationMode=disable
```

Comment:
Should be `disabled`: http://docs.deis.io/en/latest/customizing_deis/controller_settings/#changing-the-registration-mode